### ISSUE TITLE
3.0.0 causes Kotlin projects to fail compile

### ISSUE THREAD

**Issue by jeffdcamp** [afollestad/material-dialogs#1823]
**Library Version:**
3.0.0
 
**Affected Device(s):**
N/A
 
**Describe the Bug:**
Adding material-dialogs 3.0.0 library to a project with other libraries with the same module name 'library' causes the following compile error:

```
More than one file was found with OS independent path 'META-INF/library_release.kotlin_module'
```
NOTE: the 'library' in library_release.kotlin_module is the cause of this issue

Adding the following to my project would make the error go away, but would bust Kotlin function extensions (not a good option):

```
packagingOptions {
    exclude 'META-INF/main.kotlin_module'
}
```
It appears that the material-dialogs sample app was having the same issue and the above was added to line 20 of the build.gradle file (https://github.com/afollestad/material-dialogs/commit/968fd57b7e3c6a1b0801802bcc3d4394dea17e17)

NOTE: This bug was introduced AFTER 3.0.0-rc4 (when I compile with 3.0.0-rc4 there is no compile error and when I compile with 3.0.0 I get a build failure)

I'm guessing the core of the issue was when the "date-picker" library was added as a dependency to this project.

I can file another Issue/Bug with the "date-picker" library with the following fix:
To fix this properly either rename your module 'library' to something unique.  Or do what some open source Kotlin libraries projects do: Change the module name during compile by adding the following to the 'library' build.gradle file:

```
    android {
        compileOptions {
            kotlinOptions.freeCompilerArgs += ['-module-name', "$POM_ARTIFACT_ID"]
        }
    }
```

NOTE: $POM_ARTIFACT_ID should be replaced by your unique name or place the 
 following in the 'library' gradle.properties file:

    POM_ARTIFACT_ID=my-unique-name

**Stack trace / LogCat**:
```ruby
More than one file was found with OS independent path 'META-INF/library_release.kotlin_module'
```

**Comment by afollestad** [comment]
Ah good catch, I forgot you can set the module name - I'll do that

**Comment by afollestad** [comment]
Fixed in 3.0.1
